src/jQueryShim.js

```diff
@@ -72,21 +77,25 @@ const ajax = function(options) {
   return {
     abort: function(reason) {
       return request.abort(reason);
-    }
-  };
+    },
+    xhrSupported: (function() {
+      var xhrSupported = xhr();
+      return !!xhrSupported && ("withCredentials" in xhrSupported);
```

Comment:
Mostly good but this check `( "withCredentials" in xhrSupported );` should not be inside `xhrSupported` function but next to `cors:`. That's  because with this check `xhrSupported` is not only xhr support test anymore but actual cors support test hence the name doesn't fit the function anymore